.github/workflows/doc-build.yml

```diff
@@ -14,7 +14,18 @@ on:
     - cron: '0 0 * * *'
 
 jobs:
+  lint-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Check URLs
+        run: bash ./scripts/check_urls.sh
+      - name: Check Links
+        run: bash ./scripts/check_links.sh
```

Comment:
My suggestion is we can split into two jobs: check-urls and check-links. So we can run in parallel and get signals a little bit faster